framework_tvb/tvb/interfaces/web/controllers/simulator_controller.py

```diff
@@ -813,6 +825,10 @@ def set_monitor_params(self, current_monitor, **data):
             indexes = self._get_variables_of_interest_indexes(all_variables, chosen_variables)
             form = get_form_for_monitor(type(monitor))(indexes)
             form.fill_from_post(data)
+
+            if isinstance(form, SpatialAverageMonitorForm) and form.default_mask.data is None:
+                form.default_mask.data = 'hemispheres'
```

Comment:
Why this is necessary? Isn't enough to have a default in the form and this field to be requored?